src/framework/theme/components/user/_user.component.theme.scss

```diff
@@ -6,59 +6,54 @@
 
 @mixin nb-user-theme() {
   nb-user {
-
-    font-size: nb-theme(user-font-size);
-    line-height: nb-theme(user-line-height);
-
     .user-picture {
-      height: nb-theme(user-size-medium);
-      width: nb-theme(user-size-medium);
-      background: nb-theme(user-bg);
-      border: solid 2px nb-theme(user-fg-highlight);
+      background-color: nb-theme(user-picture-box-background-color);
+      border: nb-theme(user-picture-box-border-width) solid nb-theme(user-picture-box-border-color);
+    }
 
-      &.background {
-        color: nb-theme(user-fg);
-      }
+    .initials {
+      color: nb-theme(user-initials-text-color);
+      font-family: nb-theme(user-initials-text-font-family);
+      font-weight: nb-theme(user-initials-text-font-weight);
```

Comment:
line-height missing?

Reply:
Nope. Line height depends on size, so it's set below (line 40) along with font size.